cli/azd/pkg/tools/kubectl/models.go

```diff
@@ -107,7 +107,7 @@ type ServiceStatus struct {
 
 type Port struct {
 	Port       int    `json:"port"`
-	TargetPort int    `json:"targetPort"`
+	TargetPort string `json:"targetPort"`
```

Comment:
Are there cases where just a number is on the wire instead of a string representing a number?  From all the k8s docs, it looks like in most examples a raw number (not a string) is used in the manifest, so I'm not sure if we are going to run into cases where a raw number might come across.

Reply:
You are right :) I needed to update the struct to `any` (aka `interface{}`).  Added tests to confirm the unmarshalling.